### PR TITLE
Update eslint rules

### PR DESCRIPTION
**Pull request by nlaz**
This PR updates the eslint rules to follow the Airbnb style guide:`eslint-config-airbnb`. This matches with the config that we use on the streetlives-api repo with particular extensions for jsx code. The [Airbnb style guide rules](https://github.com/airbnb/javascript) are opinionated so it would be good to have some agreement on introducing these changes before merging.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,23 +7,27 @@
...
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
+    "lint": "eslint .",
     "storybook": "start-storybook -p 9009 -s public",
     "build-storybook": "build-storybook -s public"
-  },
```
Comment (Rovack) [inline package.json]: Why did these all have to move to normal dependencies?

File: src/app/App.test.jsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```
Comment (Rovack) [inline src/app/App.test.jsx]: Wait, the _tests_ need to be ".jsx"? That seems like an odd rule...

File: .eslintignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+dist/*
```
Comment (Rovack) [inline .eslintignore]: Do we actually have a /dist directory here?

File: src/app/App.test.jsx (hunk 2 above)
Comment (nlaz) [reply src/app/App.test.jsx]: Yeah. Any file with JSX in it needs a `.jsx` extension:

**Pull request merged**